docs/src/routes/docs/installation.svx

```diff
@@ -2,37 +2,61 @@
 name: Installation
 ---
 
+<script>
+  import CopyableCode from 'src/components/docs/copyable-code.svelte';
+</script>
+
 # Installation
 
 **Step 1.** Install the library with NPM or Yarn:
 
-```bash
-npm install --save-dev attractions
-# -- or --
-yarn add -D attractions
-```
+<CopyableCode>
```

Comment:
There doesn't seem to be a code block that is not `<CopyableCode>`. Does that call for a layout custom component?

Reply:
Yeah, it's not possible as of now. I've filed an issue, but until then, we'll have to make do